.indip/dashboards/main.json

```
@@ -0,0 +1,587 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": "-- Grafana --",
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
+        "name": "Annotations & Alerts",
+        "target": {
+          "limit": 100,
+          "matchAny": false,
+          "tags": [],
+          "type": "dashboard"
+        },
+        "type": "dashboard"
+      }
+    ]
+  },
+  "description": "",
+  "editable": true,
+  "fiscalYearStartMonth": 0,
+  "graphTooltip": 2,
+  "iteration": 1653988890179,
+  "links": [
+    {
+      "asDropdown": true,
+      "icon": "external link",
+      "includeVars": true,
+      "keepTime": true,
+      "tags": [],
+      "targetBlank": true,
+      "title": "Afficher les saisies associées",
+      "tooltip": "",
+      "type": "link",
+      "url": "http://127.0.0.1:10530/"
```

```suggestion
      "url": "http://127.0.0.1:10530/_/indip/input_list/from_grafana?timeRange=${__from:date}/${__to:date}"
```
https://github.com/gammadia/dsi/pull/252